### DISCUSSION TITLE
pls delete

### DISCUSSION THREAD

**Discussion by logless** [vercel/community#3878] [Account Issue]

**Comment by amyegan** [comment]
Sorry you're having trouble logging into your account, @logless. If your GitHub account was linked, you may be able to sign in that way instead of using email.

Due to the sensitive nature of account information, we cannot safely discuss it in this public forum to verify your ownership of the account. Please email [registration@vercel.com](mailto:registration@vercel.com) and include as much information as you can about the account (e.g. email, linked GitHub account, projects). That will open a support ticket with the team that can help with this 🙇

**Comment by sinisterstumble** [comment]
@amyegan please take a look at the case 00167508
